README.md

```diff
@@ -10,6 +12,17 @@
 [![Slack](https://img.shields.io/badge/%23eventing-white.svg?logo=slack&color=522a5e)](https://knative.slack.com/archives/C9JP909F0)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/5913/badge)](https://bestpractices.coreinfrastructure.org/projects/5913)
 
+## What is Knative Eventing?
```

Comment:
Is it worth mentioning in here that Eventing can be used separately from Serving/that producers and consumers don't need to be a Knative service?

Reply:
Do we say that they are requiring Knative service?

Or do you mean, we should explictly say, something like "works without serving, but also with" ?
(where the focus on the does NOT require serving is?)

Reply:
I think the second, maybe something like "event producers and consumers can be knative services, or can also work without knative serving with traditional applications". WDYT?